auth/auth_test.go

```diff
@@ -22,16 +22,14 @@ import (
 	ch "github.com/couchbase/sync_gateway/channels"
 )
 
-//const kTestURL = "http://localhost:8091"
-const kTestURL = "walrus:"
-
 var gTestBucket base.Bucket
 
 func init() {
 	var err error
 	gTestBucket, err = base.GetBucket(base.BucketSpec{
-		Server:     kTestURL,
-		BucketName: "sync_gateway_tests"}, nil)
+		Server:          base.UnitTestUrl(),
+		CouchbaseDriver: base.DefaultDriverForBucketType[base.DataBucket],
```

Comment:
No, you're right - I don't see a way to do it without adding complexity or obscuring what's going on.  Better to leave it explicit (as-is).